meetings/2022-11-09.md

```diff
@@ -47,43 +63,54 @@
+* EP: I think a good starting point in solid protocol, the resource server matching is not explicit. My implementations have to come with conformance tools to apply to both.
+* SC: Alot of things were a work in progress, part of it comes down to committing to canonical or term, if you are refering to product in another spec, as an editor, you have license to be creative about it, use language, if not too creative, use terminology from spec itself, you can distinguish solid protocol server from another server from another spec, by linking to it in terminology section. YOu can experiment, but solid server in terminology spec refers to solid protocol server, so there is consistency in that document. Perhaps that comes closer to what you are asking about.
+* EP: It brings to my mind the inhertiance v composition discussion. Solid OIDC could define a subclass of sold server which also conforms to Solid-OIDC Resource Server class, or  solid protocol vX.X defines that solid server has to implement Solid-OIDC vX.X RS,  how we are coupling those specs?
+* SC: Can I ask that we continue this discussion. These distinctions that you want to make, if we can develop some conventions in our own specs, we can go from there. Some documents we may need to do it differently than other documents. Interplay between how we want to do the tests. Let's continue this.
+* LD: Can you put your links in the notes?
+* SC: They are in the PRs. 
+* LD: I see, thank you.
+* SC: I know it is overwhelming. Let's continue to have chat discussion as to what is where and if you are missing something. Hoping we can lean on W3C work. There is alot we can do with basic linked data and spec terms, if we can take advantage of it, that would be great cause alot of units of info in our specs. When we are going into a working group, the work we are doing so far, although in a community group, it's a long prep of the group to be comfortable with language and practices of working group so we are comfortable. No demand that things are machine readable, but when we propose specs transition from working draft to proposed recommendation, the more we check these boxes, the more templates we use for conformance clauses, the better it looks and the easier to show we've done our homework. To communicate to a human we don't need that, but it looks and is more attractive and shows we've done our homework. Let's continue in the chats. We do need to work with test suite team. I'll drop a link: https://github.com/solid/process/blob/main/test-suite-panel-charter.md . There is a charter. It gives an outline of where we are going with some of that work. We have the specs machine readable, policies and procedures for authoring the tests and testing and implementation reports. So at some point we need to show interoperable implementations. And we have adequate implementation of each feature. And what classes of products, is it a data model spec or protocol spec and that helps to write the spec.  
```

Comment:
```suggestion
* SC: I know it is overwhelming. Let's continue to have chat discussion as to what is where and if you are missing something. Hoping we can lean on W3C work. There is a lot we can do with basic linked data and spec terms. If we can take advantage of it, that would be great because a lot of units of info in our specs. When we are going into a working group, the work we are doing so far, although in a community group, it's a long prep of the group to be comfortable with language and practices of working group so we are comfortable. No demand that things are machine readable, but when we propose specs transition from working draft to proposed recommendation, the more we check these boxes, the more templates we use for conformance clauses, the better it looks and the easier to show we've done our homework. To communicate to a human, we don't need that, but it is more attractive and shows we've done our homework. Let's continue in the chats. We do need to work with [test suite team](https://github.com/solid/process/blob/main/test-suite-panel-charter.md). There is a charter. It gives an outline of where we are going with some of that work. We have the specs machine readable, policies and procedures for authoring the tests, and testing and implementation reports. So at some point we need to show interoperable implementations, that we have adequate implementation of each feature, and what classes of products — is it a data model spec, or protocol spec? — and that helps to write the spec.  
```